integration_test/test_hyperopt.py

```diff
@@ -0,0 +1,100 @@
+from hyperopt import fmin, tpe, hp, STATUS_OK, STATUS_FAIL, Trials
+import hyperopt
+from sigopt.hyperopt import SigOptTrials, upload_trials
+import sigopt
+import numpy as np
+import pytest
+
+def objective(p, threshold=0.8):
```

Comment:
let's have a test with `successful_objective` and one that crashes

Reply:
done.